recipes/newrelic/apm/php/debian.yml

```diff
@@ -104,6 +104,11 @@ install:
             echo -e "{{.RED}}This installation recipe for the New Relic PHP agent must be run under sudo.{{.NOCOLOR}}" >> /dev/stderr
             exit 7
           fi
+        - |
+          if [ ! -d "/etc/apt/sources.list.d/" ]; then
+            echo -e "{{.RED}}Ensure /etc/apt/sources.list.d/ exists.{{.NOCOLOR}}" >> /dev/stderr
```

Comment:
This error would also occur if `apt-get` isn't installed, right?  I think we need to ensure the `apt-get is installed` check happens prior to this.